### ISSUE TITLE
`aarch64-unknown-linux-gnu` linking issue

### ISSUE THREAD

**Issue by charlesrocket** [taiki-e/setup-cross-toolchain-action#14]
Recently, I started getting errors with `aarch64-unknown-linux-gnu`. `cross` works as expected, but the action throws errors (https://github.com/charlesrocket/dorst/pull/18):

```
= note: /usr/lib/gcc-cross/aarch64-linux-gnu/11/../../../../aarch64-linux-gnu/bin/ld: cannot find -lz: No such file or directory
collect2: error: ld returned 1 exit status
```

**Comment by taiki-e** [comment]
Thanks for the report.

> Recently

Does this mean it worked in previous versions of setup-cross-toolchain-action?

**Comment by charlesrocket** [comment]
@taiki-e yep, last successful run was around 10 days ago. I thought its docker image or fresh deps in Cargo but setting up `cross` with cargo throws no issues (2nd commit in the PR above). Will do more tests with the successful deps to confirm.

**Comment by taiki-e** [comment]
Thanks. Probably this is due to `PKG_CONFIG_ALLOW_CROSS=1` environment variable added in 1.10.0 (if so I should revert it).
Could you check to see if the problem still occurs when setting `PKG_CONFIG_ALLOW_CROSS=0` environment variable?

**Comment by charlesrocket** [comment]
@taiki-e looks like this is the case, test commit is successful 😺

**Comment by taiki-e** [comment]
Thanks for confirming. I reverted the addition of `PKG_CONFIG_ALLOW_CROSS=1` and released it as 1.11.2.
